build.sh

```diff
@@ -3,6 +3,11 @@ echo "$1 ($2, $3)..."
 
 make clean 1>/dev/null 2>/dev/null
 
+if [ -f check-source.pl ] ; then
```

Comment:
I don't like the fact that all builds fail now because of a formatting error in the sources